website/src/pages/dbt-cloud/api-v1.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Layout from "@theme/Layout";
+
+import { API } from "@stoplight/elements";
+import "@stoplight/elements/styles.min.css";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+function dbtCloudAPI() {
+  return (
+    <Layout permalink="/">
+      <API
+        //   todo: update this url
+        apiDescriptionUrl="https://raw.githubusercontent.com/dbt-labs/dbt-cloud-openapi-spec/mbarker/generated-docs/openapi-v1.yaml"
```

Comment:
TODO: Note that this is pointing to a non-main branch for testing purposes but will change this once we're ready to deploy